docs/configuration.md

```diff
@@ -60,7 +60,16 @@ telemetries: [ 'errors', 'performance', 'http' ]
 ```
 ```javascript
 telemetries: [ 
-    [ 'errors', { stackTraceLength: 500 } ], 
+    [ 'errors', { 
+        stackTraceLength: 500, 
+        shouldIgnoreError: (errorEvent) => {
```

Comment:
I think based on the param name `shouldIgnoreError` and the simple function should be sufficient for how to use the feature - thoughts?